### PR TITLE
Articles

### LINKED ISSUE #69: Featured Articles - Import articles dynamically using static props

### LINKED ISSUE #70: Featured Articles - Create headless cms w/ api endpoints
Strapi or next js endpoints would work

### PR DESCRIPTION
**Pull request by LemonFace0309**
Code Sprint 1 - MVP Almost Done

closing #69 #70 
make sure to npm install before starting the server. Used react moment for article time stamps and react markdown to translate text from Strapi's rich text editor onto the articles.

image
- will increase image size for smaller screens lol
image

image
image

### CODE REVIEW

**Comment by jimmyygit** [comment]
Jihyo and feet...

File: next.config.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  images: {
+    domains: [
+      'localhost',
+      '35.182.254.224',
+      '35.182.99.194',
```
Comment (jimmyygit) [inline next.config.js:6]: Whats the difference between the two 35.xxx.... domains? just curious

File: pages/featured-articles/[slug].js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    paths,
+    fallback: false,
+  }
+}
+
+export async function getStaticProps({ params }) {
+  const articles = await fetchAPI(`/articles?slug=${params.slug}`)
+  return {
+    props: { article: articles[0] },
+  }
```
Comment (jimmyygit) [inline pages/featured-articles/[slug].js:77]: Why are you always indexing the first element in `articles`? asking cuz idk how it works

File: pages/featured-articles/index.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+import { getStrapiMedia } from '../../lib/media'
+import Banner from '../../components/shared/Banner'
+import Post from '../../components/FeaturedArticles/Post'
+import styles from '../../styles/Featured-Articles.module.css'
+
+function featuredArticles({ posts }) {
+  const router = useRouter()
+
+  const renderPost = (post) => {
+    const url = getStrapiMedia(post.image[0]) // 'http://localhost:8080/uploads/jihyo_852a4651cd.jpg'
```
Comment (jimmyygit) [inline pages/featured-articles/index.js:17]: same question here with `image[0]`. nice jihyo pic btw :)

**Review by jimmyygit — APPROVED** [review]
Solid work :)
I just had some questions about Strapi when ur loading static props.
The react stuff looks great too.

File: next.config.js (hunk 1 above)
Comment (functionstackx) [reply next.config.js:6]: It is just two different server IP. 

We initially launched one by it isn't enough power so we made another one and that got assigned a different public IP.

Comment (jimmyygit) [reply next.config.js:6]: What do you mean by "power"? what resources does it use? like cpu and memory?

Comment (LemonFace0309) [reply next.config.js:6]: strapi required that the vm has more than 1gb of ram lol. We chose the one with just 1gb at first

File: pages/featured-articles/[slug].js (hunk 2 above)
Comment (LemonFace0309) [reply pages/featured-articles/[slug].js:77]: it returns an array of all articles with the slug. We just want the first one (which is usually the only one).

**Pull request merged**